packages/ag-charts-community/src/chart/label.ts

```diff
@@ -124,6 +125,22 @@ export class Label<TParams = never, TDatum = any>
     }
 }
 
+type LabelBoxingMixin = { border?: { enabled?: boolean; stroke?: string }; fill?: unknown; padding?: Padding };
+export function expandLabelPadding(label: LabelBoxingMixin | undefined): Required<PaddingOptions> {
+    const { enabled: borderEnabled = false, stroke: borderStroke = undefined } = label?.border ?? {};
```

Comment:
```suggestion
    const { enabled: borderEnabled = false, stroke: borderStroke } = label?.border ?? {};
```
nit: A default value only ever applies when the value is undefined, so assigning it a default value of undefined is just redundant.